### ISSUE TITLE
Tags are not shown for files with one function

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ijt/emacs-nav#39]
```
What steps will reproduce the problem?
1. Locate file with one function.
2. Open it with 't' or shift-left-mouse
3. tags display is empty



```

Original issue reported on code.google.com by `matthew....@gmail.com` on 13 Aug 2009 at 5:31

**Comment by GoogleCodeExporter** [comment]
```
It also fails to do anything when there are no functions. An error appears:

Wrong type argument: stringp, nil


```

Original comment by `issac.tr...@gmail.com` on 14 Aug 2009 at 5:05

**Comment by GoogleCodeExporter** [comment]
```
When there are no variables, no "Variables" entry comes back from the imenu 
function. 
So using cddr throws out a function in that case.
```

Original comment by `issac.tr...@gmail.com` on 14 Aug 2009 at 6:14

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in 1a425ad05a.
```

Original comment by `issac.tr...@gmail.com` on 14 Aug 2009 at 6:26
- Changed state: **Fixed**
